public/index.html

```diff
@@ -134,60 +134,68 @@ <h2 class="solution__hdr">
         <h3 class="solution__section__hdr">She gets to know you.</h3>
         <p class="solution__section__content">Florence strikes up a conversation, learning details about your life, health, goals, and preferences in an engaging, human way.</p>
         <p class="solution__section__content">This conversation unfolds over time, making this feel less like twenty questions and more like the start of a relationship.</p>
-        <img class="solution__section__img1-bg" src="assets/images/section1-illo-bg.jpg" alt="background paint splash">
-        <img class="solution__section__img1" src="assets/images/section1-illo.svg" alt="mug with steam">
+        <img class="solution__section__img1-bg solution__section__bg" src="assets/images/section1-illo-bg.jpg" alt="background paint splash">
+        <img class="solution__section__img1 solution__section__img" src="assets/images/section1-illo.svg" alt="mug with steam">
       </div>
       <div class="solution__section__right">
         <figure>
           <p class="screen-reader">An example text conversation between Florence and a patient</p>
           <ol class="bubble">
             <li class="bubble__florence">Hi! I'm Florence. What's your name?</li>
-            <li class="bubble__patient">Mike</li>
+            <div class="bubble__wrapper">
+              <li class="bubble__patient">Mike</li>
+            </div>
```

Comment:
i didn't know about valid ol/ul children! flipped them per suggestion.

thanks for the explanation above. will definitely implement for any future similar styles. since the styles are pretty set I will leave as is for now and simplify my styles further using the current classes (sans wrappers for florence speech bubbles)